.prettierrc

```diff
@@ -1,6 +1,13 @@
 {
-  "trailingComma": "es5",
-  "tabWidth": 2,
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "jsxSingleQuote": false,
+  "quoteProps": "as-needed",
```

Comment:
Trying to keep this slim intentionally to minimize down what's used. Let's pull this out unless something is really important

Reply:
most of these are explicit default values. 

`singleQuote` is flipped to true, and printWidth has +20 characters. otherwise they are the prettier defaults. Do you still want them removed?